packages/test-helpers/src/asserts/assertThrow.js

```diff
@@ -84,8 +86,13 @@ async function assertRevert(blockOrPromise, expectedReason, ctx) {
   }
 
   // Truffle v5 provides `error.reason`, but v4 does not.
-  if (!error.reason && error.message.includes(THROW_ERROR_PREFIX)) {
-    error.reason = error.message.replace(THROW_ERROR_PREFIX, '').trim()
+  if (!error.reason) {
+    if(error.message.includes(THROW_ERROR_PREFIX)) {
+      error.reason = error.message.replace(THROW_ERROR_PREFIX, '').trim()
+    }
+    if(error.message.includes(THROW_ERROR_PREFIX_V4)) {
+      error.reason = error.message.replace(THROW_ERROR_PREFIX_V4, '').trim()
+    }
```

Comment:
See below my comment about Truffle 5

Reply:
So, if we are assuming that we are always using truffle 5, shouldn’t remove the `if` completely? It is supposed to be for truffle 4 (see comment above), but it was not working for me without this fix.